ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -1,23 +1,40 @@
 ---
+- block:
+  - name: Open firewall ports (iptables)
+    iptables:
+      action: insert
+      chain: INPUT
+      ctstate: NEW
+      protocol: "{{ item.protocol }}"
+      destination_port: "{{ item.port }}"
+      jump: ACCEPT
+    with_items:
+      - { port: 'http', protocol: 'tcp' }
+      - { port: 'https', protocol: 'tcp' }
+      - { port: 'amqps', protocol: 'tcp' }
+      - { port: 'amqp', protocol: 'tcp' }
 
-- name: Open firewall ports (iptables)
-  iptables:
-    action: insert
-    chain: INPUT
-    ctstate: NEW
-    protocol: "{{ item.protocol }}"
-    destination_port: "{{ item.port }}"
-    jump: ACCEPT
-  with_items:
-    - { port: 'http', protocol: 'tcp' }
-    - { port: 'https', protocol: 'tcp' }
-    - { port: 'amqps', protocol: 'tcp' }
-    - { port: 'amqp', protocol: 'tcp' }
+  - name: Install required packages for RHEL 6
+    action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
+    with_items:
+        - libselinux-python
+        - policycoreutils-python
+    notify:
+      - Save IPv4 iptables configuration
+
+  - name: Setup qpid custom repo
+    get_url: url=https://copr.fedorainfracloud.org/coprs/g/qpid/qpid/repo/epel-6/irina-qpid-epel-6.repo dest=/etc/yum.repos.d/copr-qpid.repo
+
+  - name:  Allow Apache to listen on tcp port 5000
+    seport:
+      ports: 5000
+      proto: tcp
+      setype: http_port_t
+      state: present
+    notify:
+      - Save IPv4 iptables configuration
```

Comment:
Does the `seport` module fiddle with iptables rules?

What makes this module specific to RHEL 6?